custom_model_runner/datarobot_drum/resource/predict_mixin.py

```diff
@@ -249,19 +249,42 @@ def _transform(self, logger=None):
         return response, response_status
 
     def do_predict(self, logger=None):
+        wrong_target_type_error_message = (
+            "This project has target type '{}', use the {{}} endpoint.".format(
```

Comment:
Should we refer to it as to model instead of project?

Reply:
yeah....